msvc/projectMSDL.vcxproj

```diff
@@ -205,6 +205,7 @@
   </Target>
   <Target Name="Build-Presets" BeforeTargets="PrepareForBuild">
     <Exec Command="xcopy /Y /Q /I &quot;$(MSBuildThisFileDirectory)../presets\presets_tryptonaut&quot; &quot;$(OutDir)presets&quot;" />
+    <Exec Command="xcopy /Y /Q /I &quot;$(MSBuildThisFileDirectory)../presets\textures_tryptonaut&quot; &quot;$(OutDir)textures&quot;" />
```

Comment:
any chance we could install all the presets?

Reply:
Yes of course - I was just going with what was done previously. I am not sure if there was a reason to use tryptonaut presets by default.